src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -248,6 +284,21 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		}
 	}
 
+	private scrollToActiveCell(): void {
+		const activeCellElement = document.querySelector(`.notebook-cell.active`);
+		activeCellElement.scrollIntoView({ behavior: 'smooth', block: 'nearest' });
+	}
+
+	private toggleEditMode(): void {
+		let selectedCell: TextCellComponent | CodeCellComponent = undefined;
+		if (this.model.activeCell.cellType !== CellTypes.Code) {
+			selectedCell = this.textCells.filter(c => c.cellModel.id === this.activeCellId)[0];
```

Comment:
find seems like a better thing to use here since you only expect there to be one cell